Data/Binary/Serialise/CBOR/Class.hs

```diff
@@ -193,6 +194,14 @@ instance Serialise Double where
     encode = encodeDouble
     decode = decodeDouble
 
+#if MIN_VERSION_base(4,7,0)
+-- | Values are serialised in units of least precision represented as
+--   @Integer@.
+instance HasResolution e => Serialise (Fixed e) where
+    encode (MkFixed i) = encode i
+    decode = MkFixed <$> decode
```

Comment:
Given that there are multiple possible representation here, I'm slightly worried at using a not-easily-extensible rep here. Arguably we ought to actually represent this as a CBOR ratio (which would also make it easy to support 7.6 where all we get is a `Rational`).